src/test/codegen-units/partitioning/compiler-builtins.rs

```diff
@@ -1,40 +0,0 @@
-// Verifies that during compiler_builtins compilation the codegen units are kept
-// unmerged. Even when only a single codegen unit is requested with -Ccodegen-units=1.
-//
-// compile-flags: -Zprint-mono-items=eager -Ccodegen-units=1
```

Comment:
Is it worth keeping this test with the codegen-units=1000 logic and making sure we're not merging in that case?

Reply:
I suspect not in the sense that there's already likely tests for that, and this is just falling back on existing functionality.